Platform/MeteorlakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -1131,29 +1111,30 @@ PlatformUpdateAcpiTable (
           Address = MSR_PC10_RCNTR;
           ResidencyCounterFrequency = 0; //Counter runs at TSC frequency
           UpdLpiStat((ACPI_LOW_POWER_IDLE_TABLE *)Table, 0, Flags, Address, ResidencyCounterFrequency);
-          if (IsPchP () || IsPchN ()) {
+          // if (IsPchP () || IsPchN ()) {
```

Comment:
Please remove commented out code, including line 1122 ~ 1130

Reply:
fixed in latest push